### PR TITLE
Add LARS cookbook example.

### PR DESCRIPTION
**Pull request by geektoni**
* Add Observable Models section to the cookbook index page.

### CODE REVIEW

File: doc/cookbook/source/examples/observers/least_angre_regression.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+=================================
+Observable Least Angle Regression
+=================================
+
+In this example, we will show how it is possible to obtain useful information from
```
Comment (karlnapf) [inline doc/cookbook/source/examples/observers/least_angre_regression.rst]: some grammar mistakes:
"In this example, we show how to retrieve information from a Shogun model during training"

File: doc/cookbook/source/examples/observers/least_angre_regression.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+a Shogun's model while it is training. Here we will train a Least Angle
```
Comment (karlnapf) [inline doc/cookbook/source/examples/observers/least_angre_regression.rst]: don't mix tense: "we will train" or "we first create" (below). I would just stick to present tense

File: doc/cookbook/source/examples/observers/least_angre_regression.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+.. sgexample:: least_angle_regression:preprocess_features
+
+We also create an instance of :sgclass:`CLeastAngleRegression` with some custom parameters
+which will be our model.
+
+.. sgexample:: least_angle_regression:create_instance
+
+Then we create an observer object which is attached to the LARS object. Here
+we are using the :sgclass:`CParameterObserverLogger` which basically will print
+on stdout all the information it will get from the machine during the training
```
Comment (karlnapf) [inline doc/cookbook/source/examples/observers/least_angre_regression.rst]: "which basically prints all information it gets from the machine during training on stdout"

File: doc/cookbook/source/examples/observers/least_angre_regression.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+process.
+
+.. sgexample:: least_angle_regression:create_observer
+
+Finally, we train the model and we apply it to the test features.
+
+.. sgexample:: least_angle_regression:train_and_apply
+
+From the observer, we can also extract the observations :sgclass:`ObservedValue`
+recorded during the training. Here we will extract the last observation.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/observers/least_angre_regression.rst]: remove "will"

**Comment by geektoni** [comment]
I've addressed your comments @karlnapf :)

**Comment by karlnapf** [comment]
Merge it then :)

**Pull request merged**